docs/rules/no-implicit-globals.md

```diff
@@ -0,0 +1,40 @@
+# disallow `var` and named functions in global scope (no-implicit-globals)
+
```

Comment:
Thats unfortunate. Babel also just rewrites `const` as `var` in the top-level scope, so thats going to leak as well.